scripts/fixtures_qa/two_type.json

```diff
@@ -0,0 +1,213 @@
+[
+    {
+        "mimetype": 50, 
+        "type": {
+            "name": "Generic",
+            "major_version": 1, 
+            "minor_version": 0, 
+            "patch_version": 0
+        },
```

Comment:
Similarly in this file, let's change the `type` to `schema_name` and `schema_version`, although in this file you should create two separate type names/versions for the trades events and the document events.